docs/source/guides/getting_started/automating_downloads.rst

```diff
@@ -81,7 +81,7 @@ Docker and Unraid
 
       docker compose restart
 
-    The script that will execute automatically is located at ``/config/run-cron``. 
+    The script that will execute automatically is located at ``/config/ytdl-sub-configs/run_cron``.
```

Comment:
`/config/run_cron` is the path for headless -- doesn't have the `ytdl-sub-configs` folder

Reply:
See the `.. tab-item` part which separates GUI from headless